tests/core/test_bootstrap_organization.py

```diff
@@ -3,20 +3,18 @@
 
 import oscrypto
 import pytest
-from quart import Response
 
 from parsec.api.data import EntryName
 from parsec.api.protocol import DeviceLabel, HumanHandle, OrganizationID, UserProfile
 from parsec.core.fs.storage.user_storage import user_storage_non_speculative_init
 from parsec.core.invite import InviteAlreadyUsedError, InviteNotFoundError, bootstrap_organization
 from parsec.core.types import BackendOrganizationBootstrapAddr
 from parsec.sequester_crypto import SequesterVerifyKeyDer, sequester_authority_sign
-from parsec.serde import packb
 
 
 @pytest.mark.trio
 @pytest.mark.parametrize("with_labels", [False, True])
-@pytest.mark.parametrize("backend_version", ["2.5", "2.6", "latest"])
+@pytest.mark.parametrize("backend_version", ["latest"])
```

Comment:
You can remove this parametrize decorator now that there is only one choice ^^

Reply:
:+1: